client/main.go

```diff
@@ -275,7 +275,7 @@ func claim(ctx *cli.Context) error {
 		return err
 	}
 
-	txID, err := arkSdkClient.Claim(ctx.Context)
+	txID, err := arkSdkClient.SendOffChain(ctx.Context, nil, nil)
```

Comment:
we kept CLI cmd name ‘claim’, although we we removed it from SDK and referenced it in cmd(SendOffchain).

Reply:
+1 for settle cmd